### PR TITLE
Dec21 probs

### PR DESCRIPTION
**Pull request by bqi343**
Based off #902

_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: content/3_Silver/Greedy_Sorting.problems.json [hunk 1]
```diff
@@ -213,6 +213,19 @@
         "kind": "autogen-label-from-site",
         "site": "CF"
       }
+    },
+    {
+      "uniqueId": "usaco-1158",
+      "name": "Closest Cow Wins",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1158",
+      "source": "Silver",
+      "difficulty": "Hard",
```
Comment (jessechoe10) [inline content/3_Silver/Greedy_Sorting.problems.json:222]:
```suggestion
      "difficulty": "Very Hard",
```

**Comment by bqi343** [comment]
Some notes:
 - Marked "Closest Cow Wins" as "Hard" instead of "Very Hard" in the module since it seemed weird to jump from "Normal" to "Very Hard." Plus, it's not really that much harder than the other problems in the module.
 - Not planning on adding "Connecting Two Barns" or "Convoluted Intervals" to the corresponding modules unless someone can make a convincing case that they should be.

File: content/extraProblems.json [hunk 2]
```diff
@@ -472,6 +472,110 @@
...
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Greedy"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1156"
+      }
+    },
+    {
+      "uniqueId": "usaco-1159",
```
Comment (jessechoe10) [inline content/extraProblems.json:489]: If you don't want to add this to DFS module, maybe add it to the conclusion problems module

File: content/extraProblems.json [hunk 3]
```diff
@@ -472,6 +472,110 @@
...
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Connected Components"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1159"
+      }
+    },
+    {
+      "uniqueId": "usaco-1160",
```
Comment (jessechoe10) [inline content/extraProblems.json:502]: Maybe add this to module. It wouldn't hurt to add a "hard" prefix sums problem to the module (consider there is only one).

**Review by jessechoe10 — CHANGES_REQUESTED** [review]

**Comment by jessechoe10** [comment]
Also, why is Closest Cow wins the only problem with a module. Do these problems not teach something new lol.

File: content/extraProblems.json (hunk 2 above)
Comment (bqi343) [reply content/extraProblems.json:489]: Can't recall having ever added anything to the conclusion problems, you can do in separate PR if you want.

File: content/extraProblems.json (hunk 3 above)
Comment (bqi343) [reply content/extraProblems.json:502]: From DM: I don't think convoluted intervals teaches you anything new about prefix sums. The point was just that you should use M being small somehow

**Pull request merged**